.codecov.yml

```diff
@@ -3,20 +3,12 @@ coverage:
     round: down
     range: "70...100"
 
-    status:
-        project:
-            default: on
-        patch:
-            default: off
-        changes:
-            default: off
-
 comment:
     layout: "diff, reach, files, footer"
     behavior: default
-    require_changes: no
-    require_base: no
-    require_head: yes
```

Comment:
This syntax was also incorrect.

Reply:
I think yes & no are [valid bools in yaml](https://yaml.org/type/bool.html), but I also prefer true/false 😅